Gopkg.lock

```diff
@@ -66,12 +66,9 @@
   digest = "1:b001b57f383ecbebb54c3916ca6b8be129a6486d3dbf2dfc722cbb0a0b2a8679"
   name = "golang.org/x/net"
   packages = ["context"]
-<<<<<<< HEAD
   pruneopts = ""
   revision = "9b4f9f5ad5197c79fd623a3638e70d8b26cef344"
```

Comment:
One of these revisions needs to go away. Which one does the digest correspond to?

Reply:
Ah yes, sorry, don't know how I forgot when I went to actually do the deletions. I am removing the c44.... one.